ols/src/query_helpers/question_validator.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
             "min_new_tokens": 1,
             "max_new_tokens": 4,
         }
-        super().__init__(*args, **kwargs, llm_params=llm_params)
+        super().__init__(*args, **dict(kwargs, llm_params=llm_params))
```

Comment:
so i guess my question is, where did we have code that was trying to pass the value in explicitly?

Reply:
QuestionValidator is constructed from ols/app/endpoints/ols.py if it's the question

Reply:
that is only part of my question.

my question is, who was invoking questionvalidator and passing in an llm_params arg?  because that's the only way you get the duplicate value warning/error, right?

Reply:
for you to get the warning you got, kwargs had to contain an llm_params keyword, in addition to the one we were passing to the super init explicitly.

Reply:
then why would we get the runtime error you reported?